Balancer-Bots/Swap-FlashLoan-Fee-Change/src/agent.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { AgentConfig } from "./utils";
-import { TestBlockEvent, MockEthersProvider, createAddress } from "forta-agent-tools/lib/tests";
-import { ethers, Finding, FindingSeverity, FindingType, HandleBlock, Network } from "forta-agent";
-import { provideHandleBlock } from "./agent";
+import { TestTransactionEvent, MockEthersProvider, createAddress } from "forta-agent-tools/lib/tests";
```

Comment:
```suggestion
import { TestTransactionEvent, createAddress } from "forta-agent-tools/lib/tests";
```

The providers are now unused